Private/Utilities.ps1

```diff
@@ -34,17 +34,31 @@ url = https://github.com/3shapeAS/dockerbuild-pwsh.git
     $configData | Out-File -FilePath "$dotGitPath/config" -Encoding ascii
 }
 
-function Add-Postfix {
+function Add-RegistryPostfix {
     param (
-        [Parameter(Mandatory=$true)]
-        [String] $Data,
+        [String] $Registry,
         [ValidateNotNullOrEmpty()]
         [String] $Postfix = '/'
     )
 
-    $trimmedData = $Data.Trim()
-    if ( -Not $trimmedData.EndsWith($Postfix) ) {
-            $trimmedData += $Postfix
+    # Do nothing if $Data is empty / writespace
+    if ([String]::IsNullOrWhiteSpace($Registry)) {
```

Comment:
Not sure why this is necessary. Shouldn't we hard fail in this case? Seems like the caller of the code made a mistake in this scenario.

Reply:
This is the part where, this special purposed has its behavior taylored specific to this scenario where i don't want it to fail.
Instead of having this part of the codes duplicated all over at places where it's used, i rather have it local to the method.